image-copy-ecr/README.md

```diff
@@ -2,6 +2,28 @@
 
 This sets up a Lambda function to listen for `registry.push` events to a private Chainguard Registry group, and mirrors those new images to a repository in Elastic Container Registry.
 
+### Usage
+
+You can use this terraform module to deploy this integration by instantiating
+it like this:
+
+```
+module "image-copy" {
+  source = "github.com/chainguard-dev/enforce-events//image-copy-ecr/iac"
+
+  # The Chainguard IAM group from which we expect to receive events.
+  # This is used to authenticate that the Chainguard events are intended
+  # for you, and not another user.
+  # Images pushed to repos under this group will be mirrored to Artifact Registry.
+  group = "<group-id>"
+
+  # This is the location in ECR where images will be mirrored.
+  # For example: pushes to cgr.dev/<group>/foo:1.2.3 will be mirrored to
+  # <account>.dkr.ecr.<region>.amazonaws.com/<dst_repo>/foo:1.2.3
+  dst_repo = "mirrored/images"
```

Comment:
so a mirror will be needed for each image in cgr.dev? since ecr doesnt work like gcr?

Reply:
You'd set this up once for the "root" ECR repo you want to mirror into, and the Lambda function will create sub-repos as necessary.

`cgr.dev/foo.com/go:1.2.3` -> `<account>.dkr.ecr.<region>.amazonaws.com/mirrored/images/go:1.2.3`
`cgr.dev/foo.com/php:1.2.3` -> `<account>.dkr.ecr.<region>.amazonaws.com/mirrored/images/php:1.2.3`

etc